wire-ios-request-strategy/Sources/Request Strategies/Conversation/ConversationEventProcessor.swift

```diff
@@ -326,7 +326,7 @@ public class ConversationEventProcessor: NSObject, ConversationEventProcessorPro
     private func updateMLSStatus(for conversation: ZMConversation, context: NSManagedObjectContext) async {
         await mlsEventProcessor.updateConversationIfNeeded(
             conversation: conversation,
-            groupID: nil,
+            groupID: .none,
```

Comment:
What is the advantage of using `.none` over `nil`?

Reply:
There isn't any. For me it just read nicer: `groupID: "none"`.
I'll revert it.